web/app/themes/clarity/inc/post-types/notes-from-antonia.php

```diff
@@ -64,8 +64,7 @@
                 'notes_from_antonia',
                 'edit.php?post_type=note-from-antonia',
                 '',
-                'dashicons-welcome-write-blog',
-                1
```

Comment:
Position 1 was removed to allow the Homepage to be positioned independently from Notes from Antonia.